gen/dcos-config.yaml

```diff
@@ -211,9 +211,57 @@ package:
       }
 {% case "false" %}
 {% endswitch %}
+  - path: /etc_master/logrotate_master.config
+    content: |
+      compress
+      delaycompress
+      notifempty
+      missingok
+      nomail
+      /var/lib/dcos/mesos/log/* {
+          olddir /var/lib/dcos/mesos/log/archive
+          maxsize 256M
+          rotate {{ mesos_log_retention_count }}
+          copytruncate
+          postrotate
+              find /var/lib/dcos/mesos/log/ /var/lib/dcos/mesos/log/archive -mtime +14 -delete
```

Comment:
I don't think this needs an update.

We introduced the `postrotate` because the `mesos-fetcher` was inheriting the agent's GLOG flags and thereby creating a bunch of empty files in the log directory every time it was called.  Over time, this cluttered the log directory (and made logrotate segfault).

With the changes to logrotation, DC/OS will only ever create one log file (`mesos-master.log` or `mesos-agent.log`) and up to `mesos_log_retention_count` files in the `archive` folder.  The `mesos-fetcher` files will be no more.

After these commits, the `postrotate` will only be there to clean up unknown files, older than two weeks, in the log directory.  These files will have no relation to the Mesos logs.